src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -715,6 +715,18 @@ public void TestProcessStartTime()
             }
         }
 
+        [Fact]
+        [PlatformSpecific(~TestPlatforms.OSX)] 
+        public void ProcessStartTime_Deterministic_Across_Instances()
+        {
+            CreateDefaultProcess();
+            for (int i = 0; i < 10; ++i)
+            {
+                var p = Process.GetProcessById(_process.Id);
```

Comment:
Nit: wrap this in a using so the Process instance is disposed once we've asserted